libs/events/import/api/src/lib/import/events-import.resolver.ts

```diff
@@ -0,0 +1,54 @@
+import {Args, Int, Mutation, Query, Resolver} from '@nestjs/graphql'
+import {
+  ImportedEventFilter,
+  ImportedEventsDocument,
+  Event,
+  SingleEventFilter,
+  CreateEventArgs
+} from './events-import.model'
+import {EventsImportService} from './events-import.service'
+
+@Resolver()
+export class EventsImportResolver {
+  constructor(private events: EventsImportService) {}
+
+  @Query(returns => ImportedEventsDocument, {
+    name: 'importedEvents',
+    description: `
+      Returns a list of imported events from external sources, transformed to match our model.
+    `
+  })
+  importedEvents(
+    @Args('filter', {nullable: true}) filter: ImportedEventFilter,
+    @Args('order', {nullable: true, type: () => Int}) order: 1 | -1,
+    @Args('skip', {nullable: true, type: () => Int}) skip: number,
+    @Args('take', {nullable: true, type: () => Int}) take: number,
+    @Args('sort', {nullable: true}) sort: string
+  ) {
+    return this.events.importedEvents({filter, order, skip, take, sort})
+  }
+
+  @Query(returns => Event, {
+    name: 'importedEvent',
+    description: `
+      Returns a more detailed version of a single importable event, by id and source (e.g. AgendaBasel).
+    `
+  })
+  importedEvent(@Args('filter') filter: SingleEventFilter) {
+    return this.events.importedEvent(filter)
+  }
+
+  /*
+  Mutations
+ */
+  @Mutation(returns => String, {
+    name: 'createEvent',
+    description: `
+      Creates and event based on data from importable events list and an id and provider.
+      Also, uploads an image to WePublish Image library.
+    `
+  })
+  createEvent(@Args('filter') filter: CreateEventArgs) {
```

Comment:
dont think `createEvent` is the most sensible name here as it's `createEventFromSource`